docs/ancestry.rst

```diff
@@ -1045,7 +1043,7 @@ event. At generation 100 and earlier, the two populations exchange migrants at a
 The resulting tree sequence has nodes on each tree at the specified census time.
 These are the nodes with IDs 8, 9, 10, 11, 12 and 13:
 
-.. code-block:: python
+.. jupyter-execute::
```

Comment:
We're displaying the same tree sequence twice now, so we can delete the image block below. Can you also delete the file ``_static/ts_with_census_nodes.svg`` please?

Reply:
Elsewhere in the docs were just using 
```
SVG(ts.draw_svg())
```
without the imports. I think we can probably just put the ``SVG(ts.draw_svg())`` at the end of the previous jupyter block too, there's no need for a separate box now.